### PR TITLE
New colors

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
PR #37628 gets a weird browser crash in drone CI webUI tests.

I will add the changes there one-by-one to this PR and see when it goes wrong.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by ChrisEdS** [comment]
Thanks for your help @phil-davis

File: apps/dav/lib/HookManager.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function firstLogin(IUser $user = null) {
 				try {
 					$this->calDav->createCalendar($principal, 'personal', [
 						'{DAV:}displayname' => $this->l10n->t('Personal'),
-						'{http://apple.com/ns/ical/}calendar-color' => '#1d2d44']);
```
[the remark this answers is not in the document]
Comment (ChrisEdS) [reply apps/dav/lib/HookManager.php:135]: I searched and replaced the previously used color project wide. Must be a hell of a coincidence.

[the remark this answers is not in the document]
Comment (phil-davis) [reply apps/dav/lib/HookManager.php:135]: I have no idea about that. I am just trying to find which thing crashes the UI tests.
And I guess that @ChrisEdS did a global search-replace of this color code.

[the remark this answers is not in the document]
Comment (ChrisEdS) [reply apps/dav/lib/HookManager.php:135]: Correct.

**Comment by phil-davis** [comment]
All the ordinary color code changes pass fine.
I added the new version of `core/img/background.jpg` - let's see what happens to CI for that.

**Comment by phil-davis** [comment]
The `core/img/background.jpg` from the original PR crashes the CI.
The `core/img/background.jpg` that is 1923x1280 pixels works.

**Comment by phil-davis** [comment]
See PR #37650 for this code with the commits squashed... for review.

**Comment by phil-davis** [comment]
Merged in #37650

**Pull request closed without merging**